docs/introduction/introduction.md

```diff
@@ -259,7 +259,7 @@ As a note, all the links are expressed under an `href` field, which signifies th
 }
 ```
 
-#### System-specific fields
+##### System-specific fields
```

Comment:
what is the difference between `submissionDate` and `firstDcpVersion`?

Reply:
It seems that `submissionDate` is the same as `firstDcpVersion`
and that `dcpVersion` is the same as `contentLastModified` 

It's likely that this was not always the case.

Also, possibly useful sidebar:
`contentLastModified` is not the same as `lastUpdated` because `contentLastModified` is only reflective if any of the `content` changes, and also will update the `contentLastModified` of a containing item. For example updating a biomaterial will trigger an update to the `contentLastModified` of any linked submissionEnvelope / project.